### PR TITLE
API: How to access elements and metadata

### PR DESCRIPTION
**Pull request by Paul-Cornell**
This PR attempts to address a basic question coming in a lot through the AI search component in the docs. 

See https://unstructured-53-docs-47-get-elements.mintlify.app/api-reference/how-to/get-elements

### CODE REVIEW

File: api-reference/how-to/get-elements.mdx [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+        )
+
+        # ...
+        ```
+
+        You can use standard Python [list operations](https://docs.python.org/3/tutorial/datastructures.html#more-on-lists) on this list.
+
+        You can also use standard Python [looping techniques](https://docs.python.org/3/tutorial/datastructures.html#looping-techniques) on this list to access each element in this list. 
+
+        For each individual element, you can call the following:
```
Comment (scanny) [inline api-reference/how-to/get-elements.mdx:154]: Some of these are _attributes_ and some are _methods_. 

"call" only really applies to methods.

Maybe something like:
> Each individual element has the following attributes: ...
> - `.category` provides the element's type ...
> ...
>
> In addition, the following methods are available ...
> - `.convert_coordinates_to_new_system ()` ...

I find it expressive to include the dot, indicating these are _members_, accessed like `element.category`, and to add the parentheses to distinguish methods from attributes. This is suggestive of how they are used in code and seems more clear to me.

File: api-reference/how-to/get-elements.mdx [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+
+        - `category` to get the element's `type` field value as a `str`. See [Element type](/open-source/concepts/document-elements#element-type).
+        - `convert_coordinates_to_new_system` to convert the element's location coordinates, if any, to a new coordinate system. See [Element's coordinates](/open-source/concepts/document-elements#elements-coordinates).
+        - `id` to get the element's `element_id` value as a `str`. See [Element ID](/open-source/concepts/document-elements#element-id).
+        - `id_to_hash(sequence_number)` to calculate and assign a deterministic hash as an ID. The hash ID is based on element's text, sequence number on page, page number and its filename.
```
Comment (scanny) [inline api-reference/how-to/get-elements.mdx:159]: this one `.id_to_hash()` is really for internal use only so shouldn't appear in this list.

File: api-reference/how-to/get-elements.mdx [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+        - `metadata` to get the element's `metadata` field as an `ElementMetadata` object. See [Metadata](/open-source/concepts/document-elements#metadata).
+        - `text` to get the element's `text` field value as a `str`. See [Element example](/open-source/concepts/document-elements#element-example).
```
Comment (scanny) [inline api-reference/how-to/get-elements.mdx:161]: I'm very much an alphabetical order kind of guy, however for an element, `.text` and `.metadata` are way more commonly used than the others, so consider putting those first, in that order.

File: api-reference/how-to/get-elements.mdx [hunk 4]
```diff
@@ -0,0 +1,219 @@
...
+            api_url=os.getenv("UNSTRUCTURED_API_URL")
+        )
+
+        element_dicts = [element.to_dict() for element in elements]
+        print(element_dicts)
+
+        # ...
+        ```
+
+        To serialize this list as JSON, you can use the standard [json](https://docs.python.org/3/library/json.html#module-json) `dumps` method to serialize it to JSON-formatted string or the `dump` method to save it as a JSON-formatted stream to a file. For example:
```
Comment (scanny) [inline api-reference/how-to/get-elements.mdx:200]: Probably better to suggest the `unstructured.staging.base.elements_to_json()` utility function for this job as it takes care of limiting float precision and a few other details.

**Review by scanny — COMMENTED** [review]
A few suggested changes.

**Review by MKhalusova — APPROVED** [review]
I've got nothing to add to Steve's comments. Once those are addressed, go ahead and merge

**Pull request merged**